### PR TITLE
[IOTDB-23] Add monitor statistics of file size. Phase 1.1

### PR DESCRIPTION
**Pull request by liuruiyiyang**
1. Add some internal statistics for different dir file size monitoring.

2. Unlike writing points number stats, currently these newly added stats are stored permanently.

### CODE REVIEW

File: iotdb/src/test/java/org/apache/iotdb/db/monitor/collector/FileSizeTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+public class FileSizeTest {
+
+  private static final String TEST_FILE_CONTENT = "FileSize UT test file";
+  private static final String TEST_FILE_PATH =
+      FileSizeConstants.DATA.getPath() + File.separatorChar + "testFile";
+
+  @Before
+  public void setUp() throws Exception {
+    EnvironmentUtils.closeMemControl();
```
Comment (MyXOF) [inline iotdb/src/test/java/org/apache/iotdb/db/monitor/collector/FileSizeTest.java]: Is this `closeMemControl();` necessary?

File: iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeManager.java [hunk 2]
```diff
@@ -592,40 +625,6 @@ public void deleteOverflow(String deviceId, String measurementId, long timestamp
...
-        processorIterator.remove();
-      } catch (ProcessorException e) {
-        LOGGER.error("Delete the filenode processor {} by iterator error.", processorName, e);
-        throw new FileNodeManagerException(e);
-      }
-    } finally {
-      processor.writeUnlock();
-    }
-  }
-
```
Comment (MyXOF) [inline iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeManager.java]: It seem that you move this code block to above?

File: iotdb/src/test/java/org/apache/iotdb/db/monitor/collector/FileSizeTest.java (hunk 1 above)
Comment (liuruiyiyang) [reply iotdb/src/test/java/org/apache/iotdb/db/monitor/collector/FileSizeTest.java]: not really necessary. I will have it removed.

File: iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeManager.java (hunk 2 above)
Comment (liuruiyiyang) [reply iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeManager.java]: yes, to solve a check-style issue that overload method should be together.

**Comment by liuruiyiyang** [comment]
Please don't merge this PR yet, I want to have several modification about the interface of IStatistics if possible.

**Comment by liuruiyiyang** [comment]
I will consider jiangtian’s comments when refactoring the module, appreciate the questions and suggestions.

**Pull request merged**